src/main/java/com/ioet/bpm/skills/domain/Category.java

```diff
@@ -0,0 +1,19 @@
+package com.ioet.bpm.skills.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@AllArgsConstructor
+public class Category {
+    @NotBlank
+    String name;
+    @NotBlank
+    Double businessValue;
+    @NotBlank
+    Double predictiveValue;
+    @NotBlank
+    String Id;
```

Comment:
Don't we need an additional annotation to mark this field as Pk?

Reply:
Done